flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1254,6 +1258,31 @@ class FMCMainDisplay extends BaseAirliners {
                 SimVar.SetSimVarValue("K:TOGGLE_GPS_DRIVES_NAV1", "Bool", 0);
             }
         }
+
+        this.backupNavTuning = SimVar.GetSimVarValue("L:A32NX_RMP_L_NAV_BUTTON_SELECTED", "Bool")
+            || SimVar.GetSimVarValue("L:A32NX_RMP_R_NAV_BUTTON_SELECTED", "Bool");
+
+        // Cannot be manual if RMP tuned. It erases everything
+        if (this.backupNavTuning && this.manualNavTuning) {
+            this.vor1IdIsPilotEntered = false;
+            this.vor1FreqIsPilotEntered = false;
+            this.vor2IdIsPilotEntered = false;
+            this.vor2FreqIsPilotEntered = false;
+            this._ilsFrequencyPilotEntered = false;
+            this._ilsIdentPilotEntered = false;
+            this.adf1IdIsPilotEntered = false;
+            this.adf1FreqIsPilotEntered = false;
+            this.adf2IdIsPilotEntered = false;
+            this.adf2FreqIsPilotEntered = false;
+
+            this.manualNavTuning = false;
```

Comment:
All this should not be erased... once the RMP nav is switched off the FMS should re-tune in the last state it was in.

Reply:
Hey Julian asked me to verify, we’ve tried it on the aircraft and any manually tuned beacons before the NAV switch was pressed were not returned to the manual tuning when it was deselected again?

I can always test it again, you got a reference for it keeping the manual tunes from a NAV deselect?

Reply:
Auto tuning should be kept but manual tunes I couldn’t get it to hold in the “memory (?)” if that’s the right word or not

Reply:
For the record... we discovered the Thales FMS retains the data and immediately re-tunes upon de-activation of remote tuning, while the Honeywell FMS erases the data and starts from scratch upon de-activation of remote tuning.